website/siteConfig.js

```diff
@@ -39,9 +39,9 @@ const siteConfig = {
 
   // For no header links in the top nav bar -> headerLinks: [],
   headerLinks: [
-    {doc: 'readme', label: 'Docs'},
+    {doc: 'en/contributing/install', label: 'Documentation'},
     {href: "https://github.com/microsoft/fast-dna", label: "GitHub"},
-    {href: "https://msft-docs.fast-dna.net", label: "Microsoft components"}
+    {href: "https://msft-docs.fast-dna.net", label: "Components"}
```

Comment:
Well, a few thoughts on this: a) we don't have more now so it seems overly verbose. b) if we end up with more, we would not just add them to the top bar so we would need some other way to index the various component libs (so it will need to change anyway. c) Microsoft components is not entirely accurate, more like "Microsoft styled components". Keeping "components" for now and we can evolve it as we actually have the issue of more versions.